### ISSUE TITLE
Fluxion Connection Fails Upon Launching Web Interface (Last Step)

### ISSUE THREAD

**Issue by defatpm** [deltaxflux/fluxion#397]
### Which version of fluxion are you using?
0.24

### What distribution of Linux (including the version) are you running it on?
kali-rolling
Version=2016.2

### What wireless adapter are you using? Exact model and chipset, statements like "internal" and "was working before" are not helpful at all.
Adapter: Panda Wireless PAU09
Chipset: Ralink RT2870/3070

### Which is the driver for it?
Driver=rt2800usb

### Does it support injection ( output of aireplay-ng -9)
"Injection is working!"

### General description of your issue along with the steps to reproduce it
Resolved Issue #376 . . . however, the connection now drops after running the Fake AP. 

After reading Issue #203 and #245,  I was able to make Fluxion work a few times after commenting out Lines 1581-1583, but then was stuck with "WLAN1 is Down" and no progress after choosing and launching the web interface. And export KEEP_NETWORK=1 did not have any effect at this point.

My current interfaces file reads:

`auto lo
iface lo inet loopback

auto eth0
allow-hotplug eth0
iface eth0 inet dhcp

auto wlan0
allow-hotplug wlan0
iface wlan0 inet manual
wpa-roam /etc/wpa_supplicant/wpa_supplicant.conf
iface home inet dhcp
iface mobile inet dhcp

iface wlan1 inet manual
iface wlan1mon inet manual
iface mon0 inet manual
iface mon1 inet manual`

Thanks!!

**Comment by spreedated** [comment]
try using the standard interface file

`auto lo
iface lo inet loopback

auto eth0
iface eth0 inet6 auto
iface eth0 inet dhcp`

**Comment by defatpm** [comment]
Will that kill the ability to connect to a saved WiFi on boot? Fluxion runs fine with my interface file so long as it is on the host machine. My issue only arises via VNC connection. The VNC connection is maintained during the WiFi scan phase as well.

**Comment by spreedated** [comment]
I don't know your exact setup, usually kali doesn't use the interface file to map connections, afaik.
Does your VNC conn runs through wifi or lan? - on the startup of fluxion all wifi connections are set to monitor devices, which stops the actual wifi capability, at least on my machine.
However, whats the point in using VNC anyway?

**Comment by deltaxflux** [comment]
Try to use keep network setting

**Comment by deltaxflux** [comment]
Export KEEP_NETWORK=1

**Comment by defatpm** [comment]
The intended purpose of VNC is to run Kali headless on a Raspberry Pi. Then, I can access remotely via laptop or mobile device. At boot, I am able to SSH via the two saved WiFi networks and run Fluxion or whatever else.

The WiFi connection for the VNC has no problem until the very last step. Particularly, the "Export KEEP_NETWORK=1" command solved the issue when Fluxion killed the VNC connection (wlan0) at the channel can selection step (although, I am unsure whether command is supposed to be ran each time; and whether KEEP_NETWORK=1 corresponds to wlan1 or the "first" adapter option in Fluxion).

Before adding the additional lines to /etc/network/interfaces, Fluxion ran completely. However, wlan1 (the monitor interface) was listed as down and never initiated the Fake AP.

**Comment by strasharo** [comment]
tl;dr which is the interface you're using for internet connectivity and which one is for fluxion?

**Comment by defatpm** [comment]
wlan0: internet (built in rpi3 chip)

wlan1: Fluxion (Ralink RT2870/3070 chipset)

**Comment by deltaxflux** [comment]
Seems like your wireless card is supported, do you have tried to use it without a vnc connection?

**Comment by defatpm** [comment]
Yes, it works perfect sans VNC.

**Comment by deltaxflux** [comment]
Sans means without? Never use vnc before but I look for a solution
